core/src/main/scala/org/apache/spark/ui/storage/StorageTab.scala

```diff
@@ -60,7 +60,7 @@ class StorageListener(storageStatusListener: StorageStatusListener) extends Bloc
 
   override def onStageSubmitted(stageSubmitted: SparkListenerStageSubmitted): Unit = synchronized {
     val rddInfos = stageSubmitted.stageInfo.rddInfos
-    rddInfos.foreach { info => _rddInfoMap.getOrElseUpdate(info.id, info).name = info.name }
+    rddInfos.foreach { info => _rddInfoMap.getOrElseUpdate(info.id, info)}
```

Comment:
In the case  that the ID existed, the other member of the  whole info object is not updated, so why do you want to update the member name individually?